openedx_events/content_authoring/data.py

```diff
@@ -194,3 +194,19 @@ class ContentObjectData:
     """
 
     object_id = attr.ib(type=str)
+
+
+@attr.s(frozen=True)
+class Lti1p1ContentGraded:
+    """
+    Data about LTI 1.1 content object.
+
+    Arguments:
+        lms_user_id (int): database identifier of the learner's LMS user.
+        xblock_id (UsageKey): Location of the Xblock reponsible to launch the LTI content.
+        anonymous_user_id (str): Anonymous id of the Learner's LMS user
+    """
+
+    lms_user_id = attr.ib(type=int)
```

Comment:
I think there is no need to mention this is the LMS user id, there is no other user id in edx-platform. @Nekenhei